tasks/main.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- include: packages.yml
+  when: (not openssl_certs_stat.results[0].stat.exists or
+        not openssl_certs_stat.results[1].stat.exists) and
+        not openssl_node_only_ca
```

Comment:
hey @conorsch -- just realized I never merged this and need this now for something else. So the conditional logic was necessary for the include of the `signed-cert.yml` playbook to only re-run the the entire CSR generation and signature logic if there arent already certificates laid out. It's really not necessary for the installation of the packages though. So I'll strip that off and good play to move that to the top :+1: 

Sorry for the long delay buddeh.